charts/rancher-alerting-drivers/rancher-alerting-drivers/1.0.200/Chart.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   name: sachet
   repository: file://./charts/sachet
 description: The manager for third-party webhook receivers used in Prometheus Alertmanager
+icon: https://charts.rancher.io/assets/logos/alerting-drivers.svg
```

Comment:
The icon file is in this PR. 
@cbron  Is this the right link that will work after the PR is merged?